### PR TITLE
Added a command to install squashfs-tools in README.md

### PR DESCRIPTION
**Pull request by DilipRenkila**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ backend. Installing these tools adds a bit to initramfs and may slow down your
 host boot, but are needed if you'd like to use a particular backend:
 
     sudo apt-get install lvm2 thin-provisioning-tools
-    sudo apt-get install btrfs-tools
+    sudo apt-get install btrfs-tools squashfs-tools
```
Comment (stgraber) [inline README.md]: So the only part of README.md where it should be listed is the build from source part.

All other cases assume lxd was installed from a package, in which case squashfs-tools will have been brought in that way.

**Comment by stgraber** [comment]
Left one comment as to where I'd expect squashfs-tools to be mentioned in README.md.

Also, you need to sign-off your commit (see CONTRIBUTING.md) before we can merge it.

**Comment by stgraber** [comment]
Oh and when updating README.md, you may also want to list rsync in that same package list as it appears to be missing (tar, xz and acl are already there).

**Comment by DilipRenkila** [comment]
I have signed off my commits.

**Comment by techtonik** [comment]
@DilipRenkila you've committed you're IDE sources. Also, you need to squash changes.

**Comment by DilipRenkila** [comment]
I removed my IDE sources and also i have an idea to develop a clustering application for LXD based on groups. Any Suggestions are welcome.

**Comment by techtonik** [comment]
@DilipRenkila now it looks like this PR is totally messed up. )

**Comment by stgraber** [comment]
Oh wow, this is in fact very messy :)

You really want to rebase your branch on master which should result in just your one commit showing up here.

Given that your change is properly signed-off, I can also just cherry-pick your commit into my bugfix branch and close this pull request if it's easier for you.

**Pull request closed without merging**